packages/vitest/src/node/workspace/resolveWorkspace.ts

```diff
@@ -44,6 +44,9 @@ export async function resolveWorkspace(
     'bail',
     'isolate',
     'printConsoleTrace',
+    'inspect',
+    'inspectBrk',
+    'fileParallelism',
```

Comment:
We should probably pass `--browser` too, as this kind of situation is kind of weird:

![image](https://github.com/user-attachments/assets/876b93e0-040d-4ed2-b204-57d49b881aaa)

Reply:
Oh well it's not actually as simple as adding `browser` in the CLI overrides. The `resolveConfig` is overwriting `browser.enabled`.